### DISCUSSION TITLE
If I import plugins with absolute path, it doesn't work...

### DISCUSSION THREAD

**Discussion by maeng2418** [tailwindlabs/tailwindcss#12034] [Help]
This is working code.
```
import plugins from './src/plugins/index'

export default {
  content: ['./src/**/*.{js,ts,jsx,tsx,mdx}'],
  plugins: [plugins.typographyPlugin],
}
```

This is not working code.
```
import plugins from '@plugins/index'

export default {
  content: ['./src/**/*.{js,ts,jsx,tsx,mdx}'],
  plugins: [plugins.typographyPlugin],
}
```

I wrote tsconfig.json like this.
```
{
  "compilerOptions": {
    "@/*": ["src/*"],
    "paths": {
      "@plugins/*": ["src/plugins/*"]
    }
  }
}
```
Even I checked if this path is wrong. But, it was working in other... (eg. app.tsx, and other component file level)

I wrote the file from './src/plugins/index.ts'
```
import typographyPlugin from '@tailwindcss/typography'

export default { typographyPlugin }
```

<img width="1819" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/46251629/aa722d61-7766-4c98-a4d4-9d856d4b6fc6">

**Comment by wongjn** [comment]
Consider using the first version that works, `import plugins from './src/plugins/index'`. I believe that the Tailwind configuration file would not be compiled using your TypeScript configuration.

**Reply by maeng2418** [reply]
Maybe I found the reason...
tailwind.config.ts doesn't follow compilerOptions in tsconfig.json but it follows "jiti"
https://github.com/tailwindlabs/tailwindcss/issues/11097
